e2e/wdio/platform/tests/action-list-item.e2e-spec.ts

```diff
@@ -0,0 +1,46 @@
+import { webDriver } from '../../driver/wdio';
+import { ActionListItemPo } from '../pages/action-list-item.po';
+import { checkAttributeValueTrue } from '../../helper/assertion-helper';
+
+describe('Action List Item Test Suite:', function() {
```

Comment:
please use arrow functions

Reply:
@nikvalor Could you please explain why to use the arrow function here instead of function?

Reply:
To fit the same style in code. 
Arrow functions is es6 feature and we should not use deprecated stuff

Reply:
I suppose it's not deprecated, there's some small differences between arrow function and regular function. I think for most of our test specs it won't be an issue, but could be for some. Will use the regular functions for 'describe' per Anton's comments in slack.